src/Microsoft.Diagnostics.Runtime/datatarget.cs

```diff
@@ -893,6 +894,18 @@ public static DataTarget AttachToProcess(int pid, uint msecTimeout, AttachFlag a
             return dataTarget;
         }
 
+        /// <summary>
+        /// Attaches to a snapshot process (see https://msdn.microsoft.com/en-us/library/dn457825(v=vs.85).aspx).
+        /// </summary>
+        /// <param name="pid">The process ID of the process to attach to.</param>
+        /// <returns>A DataTarget instance.</returns>
+        public static DataTarget CreateSnapshotAndAttach(int pid)
+        {
+            IDataReader reader = new LiveDataReader(pid, true);
```

Comment:
Can you make this:

    IDataReader reader = new LiveDataReader(pid, createSnapshot:true);